lms/djangoapps/support/static/support/jsx/program_enrollments/inspector.jsx

```diff
@@ -2,55 +2,174 @@ import React from 'react';
-    {props.errors.map(errorItem => (
-      <StatusAlert
-        open
-        dismissible={false}
-        alertType="danger"
-        dialog={errorItem}
-      />
-    ))}
-    <div key="edX_accounts">
-      <InputText
-        name="edx_user"
-        label="edX account username or email"
-        value={(props.learnerInfo && props.learnerInfo.user && props.learnerInfo.user.username) || ''}
-      />
-    </div>
-    <div key="school_accounts">
-      <InputSelect
-        name="IdPSelect"
-        label="Learner Account Providers"
-        value="Select One"
-        options={
-          props.orgKeys
-        }
-      />
-
-      <InputText
-        name="external_user_key"
-        label="Institution user key from school. For example, GTPersonDirectoryId for GT students"
-        value={(props.learnerInfo && props.learnerInfo.user && props.learnerInfo.user.external_user_key) || ''}
-      />
+
+const renderUserSection = userObj => (
+  <div>
+    <h3>
+      edX account Info
+    </h3>
+    <div><span>Username: </span> {userObj.username}</div>
+    <div><span>Email: </span> {userObj.email}</div>
+    {userObj.external_user_key && (
+      <div><span>External User Key</span>: {userObj.external_user_key}</div>
+    )}
+    {userObj.sso ? (
+      <div>
+        <h4>Single Sign On Record:</h4>
+        <div><span>UID: </span>{userObj.sso.uid}</div>
+        <div><span>Identity Provider: </span>{userObj.sso.provider}</div>
+      </div>
+    ) : (
+      <div> There is no Single Signed On record associated with this user!</div>)}
+    <hr />
+  </div>
+);
+
+
+const renderVerificationSection = verificationStatus => (
+  <div>
+    <h3>
+      ID Verification
+    </h3>
+    <div><span>Status: </span> {verificationStatus.status}</div>
+    <div><span>Verification Error: </span> {verificationStatus.error}</div>
+    <div><span>Should Display Verification: </span> {verificationStatus.should_display}</div>
+    <div><span>Verification Expiration Date: </span>
+      {verificationStatus.verification_expiry}
     </div>
-    <Button label="Search" type="submit" className={['btn', 'btn-primary']} />
-  </form>
+    <hr />
+  </div>
+);
+
+const renderEnrollmentsSection = enrollments => (
+  <div>
+    <h3>
+      Program Enrollments
+    </h3>
+    {enrollments.map(enrollment => (
+      <div key={enrollment.program_uuid}>
+        <div>Program {enrollment.program_uuid} Enrollments</div>
+        <div> <span>Status: </span> {enrollment.status} </div>
+        <div> <span>Created: </span> {enrollment.created} </div>
+        <div> <span>Last updated: </span> {enrollment.modified} </div>
+        <div> <span>External User Key: </span> {enrollment.external_user_key} </div>
+        {enrollment.program_course_enrollments && enrollment.program_course_enrollments.map(
+          programCourseEnrollment => (
+            <div key={programCourseEnrollment.course_key}>
+              <br />
+              <h4>Course {programCourseEnrollment.course_key}</h4>
```

Comment:
Here, especially, it was hard to read the output. Here's a place where I think a margin would be really useful. In fact, since this is a nested relationship, maybe a progressively increasing margin.